Controllers/Validations.cs

```diff
@@ -11,9 +11,9 @@ public IActionResult CheckRemote(string id)
     }
 
     [HttpPost]
-    public IActionResult CheckboxRemote(bool isChecked)
+    public IActionResult CheckboxRemote(bool isChecked, bool isCheckedToo)
     {
-        return Ok(isChecked);
+        return Ok(isChecked == isCheckedToo);
```

Comment:
This is a more interesting demo of `[Remote]`, including [`AdditionalFields`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.remoteattributebase.additionalfields).